island-is/island.is
# feat(samradsgatt): Add breadcrumbs to case details page

# Add breadcrumbs to case details page

## What

Add breadcrumbs to case details page

## Why

Breadcrumbs missing.

## Screenshots / Gifs

![image](https://user-images.githubusercontent.com/106164521/212281439-1dd86679-4af0-4653-b173-7e2473abd730.png)

https://www.figma.com/file/gFSUsFCIMYWRGa94XsmhpT/DEV-Samr%C3%A1%C3%B0sg%C3%A1tt?node-id=68%3A4030&t=4bKNkomPrY9eZAVT-0

## How to test

- checkout branch feature/samradsgatt/KAM-707-breadcrumbs-a-innsidu
- run the projct locally: yarn start samradsgatt
- navigate to http://localhost:4200/samradsgatt/Cases/id
- Breadcrumbs should look like the figma (except Samradsgatt first breadcrumbs should not be there)

## Checklist:
